front-end/src/components/MainPage.jsx

```diff
@@ -19,6 +19,42 @@ class MainPage extends Component {
     store.dispatch(openAnalysisPage());
   }
 
+  downloadTranscriptTemplate = () => {
+    const element = document.createElement("a");
+    const data = {"data": [
+        {
+          "": "23606",
+          "project_id": "1",
+          "turn_id": "dd8e6fee-2329-4470-9cc0-0c50f3d40313",
+          "ts": "2022-04-20 18:01:59.863 UTC",
+          "session_id": "c1ade6a3e2d0171d8813a2279f69a97171258a6c341547919fc0a3f95909b744",
+          "platform": "prototype",
+          "interaction_metadata": "{\"end\":false,\"locale\":\"en-US\"}",
+          "trace_format": "trace",
+          "trace_type": "speak",
+          "trace_payload": "{\"type\":\"speak\",\"payload\":{\"type\":\"message\",\"message\":\"<voice name=\\\"Alexa\\\">Rerouting. ETA for first delivery is 6:20 PM.</voice>\"}}"
```

Comment:
Maybe make it more clear which elements of the payload they might want to change vs what elements are not important. For example, the type needs to be speak or something else, but I think the message could be anything. Making it clear what can/cant change to the user is really important.